### PR TITLE
🌐 [i18n-KO] Translated `reference/repository.md` to Korean

### PR DESCRIPTION
**Pull request by junejae**
# What does this PR do?

Translated the `reference/repository.md` file of the documentation to Korean.
Thank you in advance for your review.

Part of https://github.com/huggingface/huggingface_hub/issues/1626

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@jungnerd, @heuristicwave May you please review this PR?

### CODE REVIEW

File: docs/source/ko/_toctree.yml [hunk 1]
```diff
@@ -46,8 +46,8 @@
       title: (번역 중) Login and logout
     - local: in_translation
       title: (번역 중) Environment variables
-    - local: in_translation
-      title: (번역 중) Managing local and online repositories
+    - local: package_reference/repository
+      title: 로컬과 온라인 리포지토리 관리하기
```
Comment (nuatmochoi) [inline docs/source/ko/_toctree.yml:50]:
```suggestion
      title: 로컬 및 온라인 리포지토리 관리
```
본문 제목 태그(line5)와 일치하지 않아, 일치를 위해 더 자연스럽다고 생각한 쪽으로 수정해보았습니다:)

File: docs/source/ko/package_reference/repository.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+`git` 작업이 포함되거나 리포지토리에서의 협업이 중점이 될 때 권장되는 도구입니다.
+
+## 리포지토리 클래스[[the-repository-class]]
+
+[[autodoc]] Repository
+    - __init__
+    - current_branch
+    - all
+
+## 도우미 메서드[[helper-methods]]
```
Comment (harheem) [inline docs/source/ko/package_reference/repository.md:18]:
```suggestion
## 도우미 메소드[[helper-methods]]
```

File: docs/source/ko/package_reference/repository.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+[[autodoc]] huggingface_hub.repository.files_to_be_staged
+
+[[autodoc]] huggingface_hub.repository.is_tracked_upstream
+
+[[autodoc]] huggingface_hub.repository.commits_to_push
+
+## 후속 비동기 명령[[following-asynchronous-commands]]
+
+`Repository` 유틸리티는 비동기적으로 시작할 수 있는 여러 메서드를 제공합니다.
```
Comment (harheem) [inline docs/source/ko/package_reference/repository.md:36]:
```suggestion
`Repository` 유틸리티는 비동기적으로 시작할 수 있는 여러 메소드를 제공합니다.
```

File: docs/source/ko/package_reference/repository.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+- `git_push`
+- `git_pull`
+- `push_to_hub`
+- `commit` 컨텍스트 관리자
+
+이러한 비동기 메서드를 관리하는 유틸리티는 아래를 참조하세요.
```
Comment (harheem) [inline docs/source/ko/package_reference/repository.md:42]:
```suggestion
이러한 비동기 메소드를 관리하는 유틸리티는 아래를 참조하세요.
```

**Review by harheem — COMMENTED** [review]
저희 glossary에서 method를 '메소드'로 번역하고 있기에, 이에 맞게 제안을 드립니다!

**Comment by junejae** [comment]
> @junejae review 확인 부탁드립니다!

@jungnerd 변경 제안 반영하였습니다!

**Review by Wauplin — APPROVED** [review]
Then let's merge it! Thanks everyone :hugs:

**Pull request merged**